internal/plan/plan_test.go

```diff
@@ -196,7 +196,8 @@ var planTests = []planTest{{
 				Startup:  plan.StartupUnknown,
 			},
 		},
-		Checks: map[string]*plan.Check{},
+		Checks:     map[string]*plan.Check{},
+		LogTargets: map[string]*plan.LogTarget{},
```

Comment:
Can we add at least a couple of basic tests for log targets and a merging operation?